CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ CHANGELOG
   [#4141](https://github.com/pulumi/pulumi/pull/4141)
 - Add missing builtin `MapArray` to Go SDK.
   [#4144](https://github.com/pulumi/pulumi/pull/4144)
+- Add support for policy pack configuration using CLI.
```

Comment:
Since the changelog is actually used for our releases, we may want to clarify this. "Add support for enabling Policy Packs with configuration" ... or something to that effect.